libs/components/validation/src/lib/modules/url-validation/url-validation-options.ts

```diff
@@ -3,7 +3,7 @@
  */
 export interface SkyUrlValidationOptions {
   /**
-   * Indicates the ruleset to use for URL validation. Ruleset 1 uses a regular expression and ruleset 2 uses the third-party [validator.js library](https://github.com/validatorjs/validator.js/).
+   * The ruleset to use for URL validation. Ruleset 1 uses a regular expression and ruleset 2 uses the third-party [validator.js library](https://github.com/validatorjs/validator.js/).
```

Comment:
```suggestion
   * The ruleset for URL validation. Ruleset 1 uses a regular expression, and ruleset 2 uses the third-party [validator.js library](https://github.com/validatorjs/validator.js).
```
This text doesn't currently show up in the docs because the table in the Interfaces section has a header with no rows. Do we need to file a separate bug to figure out why the `rulesetVersion` property is not showing up in the table?

Also, do we need the `/` at the end of the URL that we are linking to? I checked the URL, and it works with and without that `/`.

Reply:
@blackbaud-johnly can you file this as a separate bug? It'd be an issue with docstools.

The URL can omit the trailing slash.

Reply:
I'll file an issue, but can we also commit the style/grammar changes I suggested above in this PR?

Reply:
I created https://dev.azure.com/blackbaud/Products/_workitems/edit/2440849 to capture the bug as requested.